### DISCUSSION TITLE
What Is this called?

### DISCUSSION THREAD

**Discussion by LakesideMiners** [TandoorRecipes/recipes#1066] [Q&A]
Is it called ```Django Recipes``` like the system info view says, or is it called ```Tandoor``` or ```Tandoor Recipes```?
Trying to find this out before I make changes  and open a PR as there might be a reason for it being like that.

**Comment by vabene1111** [comment]
haha yes this is confusing, i am sorry. Django Recipes or just recipes was the name of the project for multiple years before i had a real name for it. This also means that project files and everything technical is still called this way (and also stays this way). 

The project is called Tandoor now but i prefer to call it Tandoor Recipes, so both are fine. If you want to aim for consistency somewhere then use just Tandoor.
